### PR TITLE
installer certbot support

### PR DESCRIPTION
**Pull request by csweichel**
Certbot support was in the helm chart but ignored by the installer

### CODE REVIEW

**Comment by geropl** [comment]
Sorry, does not work for me locally (yet) because of permission issues.

**Comment by csweichel** [comment]
> Sorry, does not work for me locally (yet) because of permission issues.

I don't think anything changed here that would cause those issues. If they persist and are related to the installer itself, they're a different issue.

**Comment by geropl** [comment]
@csweichel Absolutely, but cannot verify/test of the changes.

**Review by geropl — APPROVED** [review]
Code looks good!

File: install/gcp-terraform/modules/helm/main.tf [hunk 1]
```diff
@@ -60,6 +60,16 @@ resource "helm_release" "gitpod" {
     value = "[]"
   }
 
+  set {
+    name = "components.proxy.certbot.enaled"
```
Comment (Bananeweizen) [inline install/gcp-terraform/modules/helm/main.tf:64]: Typo in enabled

**Pull request merged**